lookout/style/format/features.py

```diff
@@ -410,16 +414,28 @@ def _add_noops(vnodes: Sequence[VirtualNode], path: str) -> List[VirtualNode]:
                                       y=CLASS_INDEX[CLS_NOOP], path=path))
         return result
 
-    def _get_role_index(self, vnode: VirtualNode) -> int:
+    def _get_internal_type_index(self, vnode: VirtualNode) -> int:
         role_index = -1
         if vnode.node:
             role = vnode.node.internal_type
             if role in self.roles.ROLE_INDEX:
                 role_index = self.roles.ROLE_INDEX[role]
-        elif vnode.value in self.tokens.RESERVED_INDEX:
-            role_index = len(self.roles.ROLE_INDEX) + self.tokens.RESERVED_INDEX[vnode.value]
         return role_index
 
+    def _get_keyword_index(self, vnode: VirtualNode) -> int:
+        keyword_index = -1
+        if not vnode.node and vnode.value in self.tokens.RESERVED_INDEX:
+            keyword_index = self.tokens.RESERVED_INDEX[vnode.value]
+        return keyword_index
+
+    @staticmethod
+    def _get_role_indices(vnode: VirtualNode) -> int:
+        role_indices = [0] * N_ROLES
```

Comment:
pls change to `numpy.zeros`

Reply:
current logic will not work with numpy arrays: `*self._get_role_indices(left_sibling_vnode)` this is how features are given to `_get_left_sibling_features` & `_get_right_sibling_features`